### PR TITLE
Fix: verify link is valid URL

### PR DESCRIPTION
**Pull request by jorgercosta**
The link should be a valid URL

### CODE REVIEW

**Comment by josepostiga** [comment]
@jorgercosta thank you for this. Can you add some tests to confirm the issue and the enhancement you're making? :smile:

File: domains/Links/Tests/Feature/LinksStoreTest.php [hunk 1]
```diff
@@ -84,4 +84,35 @@ public function it_fails_to_store_resources_on_validation_errors(): void
...
             ]);
     }
+
+    /** @test */
+    public function it_fails_to_store_resources_with_invalid_link(): void
+    {
+
+        //assert link validation using simple string
+        $payload = [
+            'link' => "this_is_not_a_real_url",
```
Comment (josepostiga) [inline domains/Links/Tests/Feature/LinksStoreTest.php:94]: Can we make use of PHPUnit's data providers, here? It'll remove some duplicate code in this test.

**Review by josepostiga — CHANGES_REQUESTED** [review]
Thanks for submitting the requested test case. There's only one thing missing and then we can merge this :wink:

File: domains/Links/Tests/Feature/LinksStoreTest.php (hunk 1 above)
Comment (jorgercosta) [reply domains/Links/Tests/Feature/LinksStoreTest.php]: I tried using provider notation but I kept getting

The data provider specified for Domains\Links\Tests\Feature\LinksStoreTest::it_fails_to_store_resources_with_invalid_link is invalid.
InvalidArgumentException: Unknown formatter "name"

I believe it's due to that App is not bootstrapped yet as after setup.

**Pull request merged**